src/hooks/web3Context.tsx

```diff
@@ -26,6 +26,11 @@ const ALCHEMY_ID_LIST = [
   "R3yNR4xHH6R0PXAG8M1ODfIq-OHd-d3o", // this is Zayen's
   "DNj81sBwBcgdjHHBUse4naHaW82XSKtE", // this is Girth's
 ];
+
+const TEMP_ALCHEMY_IDS = [
+  "rZD4Q_qiIlewksdYFDfM3Y0mzZy-8Naf", // appleseed-temp1
```

Comment:
It may be best to remove this key since it's close to capacity. Removing it would avoid tons of errors any time this key comes up in the "intelligent load balancer".